quote_bot.py

```diff
@@ -105,16 +105,28 @@ def famouslastwords(channel, fmt):
       if line:
           ircsock.send("PRIVMSG "+ channel + " :" + line + "\n")
 
-def ircpopularity(channel, fmt):
-  args = get_text_from_formatted(fmt).split()
-  if len(args) != 2:
-   sendmsg(channel, "Sorry, only two combatants are allowed.")
-  else:
-   fighter1 = args[0]
-   fighter2 = args[1]
-   quoteaddOut = os.popen("/home/karlen/bin/ircpopularity %s %s" % (fighter1,fighter2)).read().split("\n")
-   for line in quoteaddOut:
-      if line:
+def pondareplay(channel, fmt):
+    args = get_text_from_formatted(fmt).split()
+    if len(args) == 1:
+        flw = os.popen("python /home/karlen/bin/pondareplay --random Y" ).read().split("\n")
+    elif len(args) == 1:
+        topic = args[0]
+        flw = os.popen("python /home/karlen/bin/pondareplay --search %s" % (topic)).read().split("\n")
+    for line in flw:
```

Comment:
flw could not be assigned here - either default it to an empty list or have an `else` clause which returns nothing?